features/function.yml

```
@@ -0,0 +1,5 @@
+name: "@function"
+description: "The `@function` at-rule takes arguments, such as CSS values or custom properties, and returns a CSS value, possibly based on conditional logic such as by using the `@media` at-rule."
```

```suggestion
description: "The `@function` CSS at-rule defines a custom function that takes CSS values or custom properties as arguments, and returns a CSS value. It can be based on conditional logic such as by using the `@media` at-rule."
```

I'd love to include something like "@function --name() can be invoked by calling --name()", but made up syntax is discouraged in the style guide.